scripts/pre-commit-clang-format.sh

```diff
@@ -18,7 +18,7 @@
 ##################################################################
 # SETTINGS
 # set path to clang-format binary
-CLANG_FORMAT="/usr/bin/clang-format-6.0"
+CLANG_FORMAT="/usr/bin/clang-format-7"
```

Comment:
Do we really need an absolute path here ? Depending on the plateform, this path is not always true.

Reply:
To be tested.
clang-format 6 has bad behavior on some {} and ifs ... so I switch, but it can be done locally.

Reply:
Using alternatives capabilities of packages managers (on Linux or MacOs-macport) allows to select the right version of clang for the system.
In the development guidelines, ask for clang-format being the access to clang-7 (at least) and set this script to be 
`CLANG_FORMAT='which clang-format'` (beware of quote being backquotes) and this coule be system agnostic.